code/script-else.js

```diff
@@ -0,0 +1,4 @@
+// changing the attribute href, which results in changing css file applied
```

Comment:
script is each js file ... script = program ... useless word ... 
else ... else so this means what --- play tennis?

Reply:
script-math.js renamed to rcalc-math.js also script-else.js is rcalc-personalize.js, hope is better now?